master/test/integration/api/api_checkpoints_test.go

```diff
@@ -346,28 +346,35 @@ func testGetTrialCheckpoints(
 
 }
 
-func createPrereqs(t *testing.T, db *db.PgDB) (
+func createPrereqs(t *testing.T, pgDB *db.PgDB) (
```

Comment:
This change is catching up with https://github.com/determined-ai/determined/pull/3959 and https://github.com/determined-ai/determined/pull/3946 .